master_check.do

```diff
@@ -296,11 +292,10 @@ if ${run_field_comments} {
 /* <=============== HFC 13. Output summaries for text audits ==============> */
 
 if ${run_text_audits} {
-  ipachecktextaudit ${textaudit} using "${infile}",  ///
-    saving("${textauditdb}")  ///
+  ipachecktextaudit ${textaudit} using "${textauditdb}",  ///
     media("${sctomedia}") ///
     enumerator(${enum}) ///
-    keepvars(${keep13})
+    keepvars(${keep15})

```

Comment:
Should this be 15 or 13, it's changed back and forth a few times in these changes?

Reply:
I don't see either when I run macrolist?

Reply:
Never mind that was my mistake. Fixed!